lib/src/Base/Type/MatrixImplementation.cxx

```diff
@@ -32,6 +32,7 @@
 #include "openturns/Os.hxx"
 #include "openturns/Exception.hxx"
 #include "openturns/Sample.hxx"
+#include "openturns/IdentityMatrix.hxx"
```

Comment:
Why do you include this file? It is not used anywhere

Reply:
This is true. Fixed at 9c332e51a